pkg/git/options.go

```diff
@@ -72,6 +72,13 @@ type AuthOptions struct {
 	Identity   []byte
 	KnownHosts []byte
 	CAFile     []byte
+	// TransportAuthID is a unique identifier for this set of authentication
+	// options. It's used by managed libgit2 transports to uniquely identify
+	// which credentials to use for a particular git operation, and avoid misuse
+	// of credentials in a multi tenant environment.
+	// It must be prefixed with a valid transport protocol (ssh or http) because
+	// of the way managed transports are registered and invoked.
```

Comment:
```suggestion
	// TransportAuthID is a unique identifier for this set of authentication
	// options. It's used by managed libgit2 transports to uniquely identify
	// which credentials to use for a particular Git operation, and avoid misuse
	// of credentials in a multi-tenant environment.
	// It must be prefixed with a valid transport protocol (SSH or HTTP) because
	// of the way managed transports are registered and invoked.
```

Reply:
Same thoughts as above.